lib/div-alg.gi

```diff
@@ -504,7 +506,9 @@ b:=f.2;
 g:=f/[a^A[3],b^A[4][1]*a^(-A[4][3]),b^(-1)*a*b*a^(-A[4][2])];
 fi;
 
-I:=IsomorphismSpecialPcGroup(g);
+### Change to Permutation Group for performance ###
+# I:=IsomorphismSpecialPcGroup(g);
```

Comment:
We need to stop commenting out a block of code and writing a new code below. This makes diffs unclear, whereas editing IN the code makes changes more obvious. if you need to preserve commented out code, I've rely on version control to keep its previous version.